src/__tests__/api-test.js

```diff
@@ -0,0 +1,148 @@
+const fs = require('fs')
+
+/**
+ * Import the states daily json file
+ */
+const statesDaily = JSON.parse(
+  fs.readFileSync('./public/api/v1/states/daily.json', 'utf8'),
+)
+
+/**
+ * List of states
+ */
+const uniqueStates = [...new Set(statesDaily.map(item => item.state))]
+
+/**
+ * These arrays capture the observed anomalies in cumulative testing results.
+ * Explanation:
+ * Since state websites serve as the AOR for cumulative data, their regressions
+ * result in errors in our cumulative positive/negative tests.
+ *
+ * We should verify these anomalies with QA/Data teams and then enter them here.
+ * These anomalies will be removed from the data in tests below so that these
+ * tests will pass. When new anomalies are found, they should be verified with
+ * QA/Data and only then should be entered into the arrays of allowed anomalies
+ * below.
+ */
+
+const allowedNegativeAnomalies = [
+  {
+    state: 'AK',
+    date: 20200413,
+    positive: 277,
+    negative: 7553,
+  },
+  {
+    state: 'AK',
+    date: 20200412,
+    positive: 272,
+    negative: 7766,
+  },
+  {
+    state: 'DE',
+    date: 20200410,
+    positive: 1326,
+    negative: 10415,
+  },
+  {
+    state: 'WI',
+    date: 20200330,
+    positive: 1221,
+    negative: 15856,
+  },
+  {
+    state: 'WI',
+    date: 20200329,
+    positive: 1112,
+    negative: 16550,
+  },
+  {
```

Comment:
Example of an exception I had to add today to get tests to pass.

Reply:
<img width="600" alt="Screen Shot 2020-04-17 at 5 55 21 AM" src="https://user-images.githubusercontent.com/548922/79557123-0cf07880-8070-11ea-90d3-da484e21825b.png">

Reply:
When I see these, I just search the json to find the discrepency, then check the with QA/DE or check them myself by verifying the screenshots. If everything checks out, I add the exception so that tests pass.